alerta/auth/basic_ldap.py

```diff
@@ -40,11 +48,20 @@ def login():
     except Exception as e:
         raise ApiError(str(e), 500)
 
+    # Get email address from LDAP
+    if not email_verified:
+        try:
+            ldap_result = ldap_connection.search_s(userdn, ldap.SCOPE_SUBTREE,'(objectClass=*)',['mail'])
+            email = ldap_result[0][1]['mail'][0].decode(sys.stdout.encoding)
```

Comment:
Can we make the decode here more configurable? Looks like you can set a "encoding" config parameter that can be used for decoding responses. See https://ldap3.readthedocs.io/encoding.html#encoding

Reply:
ldap3 is a different package. I can't see any option to do similar in python-ldap - this is the most generic solution I can find

Reply:
Ok sorry about that. :+1+